### DISCUSSION TITLE
Upgrade path 2.3 to 2.4 and 2.4-beta to 2.4

### DISCUSSION THREAD

**Discussion by stedaniels** [Security-Onion-Solutions/securityonion#8864] [General] [answered]
Hi,

I've watched the video announcing 2.4, and I might have missed it.  @dougburks is there an upgrade path from 2.3 to 2.4?

Will there be an upgrade path to go from a 2.4-beta to a 2.4 production, for those of us who like to live by the seat of their pants?

Thanks

**Comment by dougburks** [comment, marked as answer]
We'll provide more information when we get closer to releasing 2.4.

**Reply by argwfm** [reply]
Any updates on timing of the release or upgrade paths for v2.4?
Planning a new build and details would help w/ scheduling...

**Reply by dougburks** [reply]
You should go ahead with the latest version of 2.3 which is currently 2.3.182:
https://blog.securityonion.net/2022/11/security-onion-23182-now-available.html

We plan to release 2.3.190 next week.

We'll provide more information about 2.4 when we get closer to releasing it.

**Reply by argwfm** [reply]
Thanks much, Doug!
